assets/pages/contributions-landing/helpers/validation.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+// ----- Imports ----- //
+
+import type { Contrib, ContribError, Amount } from 'helpers/contributions';
+
+
+// ----- Functions ----- //
+
+function validateOtherAmount(amount: Amount, contrib: Contrib): ?ContribError {
+
+  if (amount.value === '') {
+    return 'noEntry';
+  }
+
+  const numericAmount = Number(amount.value);
+
+  if (numericAmount < 5 && contrib === 'RECURRING') {
```

Comment:
I'd consider pulling these out into separate constants to separate the configuration from the logic:
```
const minRecurringAmount = 5;
const minOneOffAmount = 1;
const maxContributionAmount = 2000;
```

This will also help when we start dealing with multiple currencies.